ios/FluentUI/Avatar/AvatarView.swift

```diff
@@ -423,10 +423,11 @@ open class AvatarView: UIView {
         self.secondaryText = secondaryText
         self.presence = presence
         self.fallbackImageStyle = nil
+        setupInitialsView(convertTextToInitials: convertTextToInitials)
```

Comment:
>setupInitialsView [](http://example.com/codeflow?start=8&length=17)

seems weird that we setupinitialviews then redetermine that we are going to show intials view